src/activities.ts

```diff
@@ -7,17 +7,13 @@ export async function withdraw(details: PaymentDetails): Promise<string> {
     `Withdrawing $${details.amount} from account ${details.sourceAccount}.\n\n`
   );
   const bank1 = new BankingService('bank1.example.com');
-  try {
-    const result = await bank1.withdraw(
-      details.sourceAccount,
-      details.amount,
-      details.referenceId
-    );
-    return result;
-  } catch (error) {
-    throw new Error('Unexpected error occurred');
```

Comment:
This is swallowing the actual typed errors being returned by the client methods so the typed methods which are included as `nonRetryableErrorTypes` will never be caught